resources/views/docs/_structures/match_game.md

```diff
@@ -0,0 +1,15 @@
+## MatchGame
+
+Field        | Type                     | Description
+------------ | ------------------------ | -----------
+id           | integer                  | |
+beatmap      | [Beatmap](#beatmap)      | Includes `beatmapset`.
+beatmap_id   | integer                  | |
+start_time   | [Timestamp](#timestamp)  | Uses `Z` instead of `+00:00` for timezone.
```

Comment:
I don't think this needs to be mentioned, ISO 8601 defines both and parsing ISO 8601 means you have to be able to accept either form.